evap/rewards/models.py

```diff
@@ -34,13 +33,9 @@ def can_delete(self):
         return not self.reward_point_redemptions.exists()
 
     def redemptions_by_user(self):
```

Comment:
Maybe also rename to `users_with_redeemed_points` or similar?

Reply:
+ Also where it is used :)